src/inmanta/deploy/work.py

```diff
@@ -34,6 +35,17 @@
 T = TypeVar("T", bound=tasks.Task, covariant=True)
 
 
+class TaskPriority(IntEnum):
+    TERMINATED = -1
+    USER_DEPLOY = 0
+    NEW_VERSION_DEPLOY = 1
```

Comment:
I'm inclined to swap these two (0 and 1). @wouterdb WDYT?

Reply:
Given the current api, they will both be about the exact same set of tasks, I think, so I don't think this is very important. 

Assuming it would matter, I like this order, because it gives the user control

Reply:
(Meaning that I support swapping these two)